tests/legacy-cli/e2e/tests/basic/scripts-array.ts

```diff
@@ -25,11 +25,12 @@ export default function () {
     'src/input-script.js': 'console.log(\'input-script\');',
     'src/lazy-script.js': 'console.log(\'lazy-script\');',
     'src/pre-rename-script.js': 'console.log(\'pre-rename-script\');',
-    'src/pre-rename-lazy-script.js': 'console.log(\'pre-rename-lazy-script\');'
+    'src/pre-rename-lazy-script.js': 'console.log(\'pre-rename-lazy-script\');',
   })
     .then(() => appendToFile('src/main.ts', 'import \'./string-script.js\';'))
     .then(() => updateJsonFile('angular.json', configJson => {
-      const appArchitect = configJson.projects['test-project'].targets;
+      const appArchitect = configJson.projects['test-project'].targets
```

Comment:
I figured that it would end up being changed back at some point and then everything would need to be changed yet again.  And the E2E tests will eventually be replaced so making them a little more messy didn't seem so bad.